atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -402,10 +401,9 @@ def get_build(self, metadata, worker_metadatas):
 
             extra['image'].update(get_parent_image_koji_data(self.workflow))
 
-            flatpak_source_info = get_flatpak_source_info(self.workflow)
-            if flatpak_source_info is not None:
-                compose_info = get_compose_info(self.workflow)
-                koji_metadata = compose_info.koji_metadata()
+            flatpak_compose_info = get_flatpak_compose_info(self.workflow)
+            if flatpak_compose_info is not None:
```

Comment:
any reason this isn't just `if flatpak_compose_info:` ?  the way you have it, `flakpak_compose_info = False` would pass the test, which it obviously shouldn't.  It probably can't be set to False anyway, but why the extra characters for a less clear form of the test?

Reply:
My personal thinking is that if you are expecting a variable to hold None or an object then 'if object is not None: return object.member' catches a violation of your expectation when a variable ends up with 0/''/[]/{}/False, rather than silently letting it go. But yes, it is a fair bit of verbosity to catch a rare mistake and it does seem like   disproportionate number of the uses of 'is not None' in the atomic-reactor codebase are mine :-)